molgenis_documentation/user_documentation.md

```diff
@@ -270,26 +223,350 @@ Go to the Upload menu. You now should see something like this:
 
 ![Importer first screen](images/importer_first_screen.png?raw=true, "importer")
 
-To keep it simple, all you need to do is click the 'select a file' button, select your newly made EMX file, and press the next button until it starts importing. Don't worry about all the options you are skipping, we will handle those [later in this document](#importing-advanced). After your import is done, you can view your data in the data explorer. Go there by clicking the 'Data Explorer' link in the menu.
+To keep it simple, all you need to do is click the 'select a file' button, select your newly made EMX file, and press the next button until it starts importing. Don't worry about all the options you are skipping, we will handle those [upload section](#upload). After your import is done, you can view your data in the data explorer. Go there by clicking the 'Data Explorer' link in the menu.
 
 Congratulations! You have now deployed MOLGENIS either locally or on a server, and you have made the first steps on getting your data into the MOLGENIS database. Play around a bit with the different data explorer filters to get a feel on how MOLGENIS works.
 
-Of course, simply uploading and showing data is not the only thing you can do with the MOLGENIS software. In the following [section](#from-user-to-expert), we will take you from being a simple user, and teach you on how to be an expert.
-
----
----
+Of course, simply uploading and showing data is not the only thing you can do with the MOLGENIS software. In the following MOLGENIS step-by-step section, we will take you from being a simple user, and teach you on how to be an expert.
 
-## <a name="from-user-to-expert"></a> From user to expert, MOLGENIS step-by-step
+## <a name="molgenis-step-by-step"></a> MOLGENIS step-by-step
```

Comment:
This section includes a mix of user and expert info. So naming it from user to expert is seems exaggerated